src/components/data/pagination/components/PrevPageButton/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+type PrevPageButtonProps = {
+    onClick: () => void;
+    disabled: boolean;
+};
+
+const PrevPageButton = ({ onClick, disabled }: PrevPageButtonProps) => (
+    <button onClick={onClick} disabled={disabled}>
```

Comment:
Could all these pagination buttons be made with our Button component?

Reply:
I see now that we have those kind of buttons in the design sheet, but they are not made yet